internal/app.go

```diff
@@ -0,0 +1,187 @@
+package internal
+
+import (
+	"embed"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	ics "github.com/arran4/golang-ical"
+	"github.com/gin-gonic/gin"
```

Comment:
Please separate github imports into a separate block below the stdlib imports (just in the same () but just separated by a blank line)